### ISSUE TITLE
Client crash when chaos dragon fires its projectiles

### ISSUE THREAD

**Issue by Saereth** [brandon3055/Draconic-Evolution#1260]
## Base information
* Minecraft version: 1.12.2
* Mod version: Draconic-Evolution-1.12.2-2.3.20.333-universal
* Minecraft Forge version: 14.23.5.2795
* Mod Pack: (if applicable) FTB Interactions (Closed Beta)

## Crash report
Game crashes as soon as the chaos dragon fires its projectiles, relevant crash section:
```
net.minecraftforge.fml.relauncher.ReflectionHelper$UnableToAccessFieldException: net.minecraftforge.fml.relauncher.ReflectionHelper$UnableToFindFieldException: java.lang.NoSuchFieldException: soundName
	at net.minecraftforge.fml.relauncher.ReflectionHelper.getPrivateValue(ReflectionHelper.java:210)
	at com.brandon3055.draconicevolution.lib.DESoundHandler.playSoundFromServer(DESoundHandler.java:95)
	at com.brandon3055.draconicevolution.entity.EntityGuardianProjectile.<init>(EntityGuardianProjectile.java:98)
	at com.brandon3055.draconicevolution.entity.EntityChaosGuardian.updateAttack(EntityChaosGuardian.java:689)
	at com.brandon3055.draconicevolution.entity.EntityChaosGuardian.customAIUpdate(EntityChaosGuardian.java:595)
	at com.brandon3055.draconicevolution.entity.EntityChaosGuardian.func_70636_d(EntityChaosGuardian.java:238)
	at net.minecraft.entity.EntityLivingBase.func_70071_h_(EntityLivingBase.java:2179)
	at net.minecraft.entity.EntityLiving.func_70071_h_(EntityLiving.java:295)
	at net.minecraft.world.World.func_72866_a(World.java:1993)
	at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:832)
	at net.minecraft.world.World.func_72870_g(World.java:1955)
	at net.minecraft.world.World.func_72939_s(World.java:1759)
	at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:613)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:767)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:668)
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:185)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526)
	at java.lang.Thread.run(Unknown Source)
Caused by: net.minecraftforge.fml.relauncher.ReflectionHelper$UnableToFindFieldException: java.lang.NoSuchFieldException: soundName
	at net.minecraftforge.fml.relauncher.ReflectionHelper.findField(ReflectionHelper.java:166)
	at net.minecraftforge.fml.relauncher.ReflectionHelper.getPrivateValue(ReflectionHelper.java:206)
	... 17 more
Caused by: java.lang.NoSuchFieldException: soundName
	at java.lang.Class.getDeclaredField(Unknown Source)
	at net.minecraftforge.fml.relauncher.ReflectionHelper.findField(ReflectionHelper.java:160)
	... 18 more

```

full crash log:
https://gist.github.com/Saereth/fd58222273d8fb02f2d10c274e592a05

Please note assuming the issue is a crash something like this

```Error: net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from Some Mod```

Is NOT a crash report!</br>
I need the full crash report which you can find in the crash-reports folder in the minecraft folder.</br>
Failing that please give me your entire minecraft log from the session when the crash occurred.</br>
If you do not provide these your issue will be insta-closed.

**Comment by brandon3055** [comment]
> * Mod version: Draconic-Evolution-1.12.2-2.3.20.333-universal

I suggest you double check that. According to your log your running 2.3.19
```
| LCHIJA | draconicevolution                    | 2.3.19                    | Draconic-Evolution-1.12.2-2.3.19.330-universal.jar
```
This issue should be fixed in 2.3.20

**Comment by Saereth** [comment]
yeah weird, twitch launcher was showing me on 333 but the physical file was not 333, deleted and trying 333 right now

**Comment by Saereth** [comment]
can confirm this is good in 333, ty

**Comment by RonRicoSuave** [comment]
I think there was a temporary file screwup on curseforge, because I have a file date of late feb 13th when it showed a new version (dislike twitch and manually update when stuff is available), but the download ended up being 2.3.19.330 as above.  Just leaving this here in case anyone else hits it, version of file from curseforge is correct now.   Cheers

**Comment by FoxMcloud5655** [comment]
I had an issue a while back that prevented me from uploading a file onto CurseForge; they said they were having troubles.  I'm guessing that's what this was in your case, RonRicoSuave.
